<!DOCTYPE html>
<html lang="en">
    <head>
        <!-- Global site tag (gtag.js) - Google Analytics -->
        <script async src="https://www.googletagmanager.com/gtag/js?id=G-DWKX9XJ8QW"></script>
        <script>
            window.dataLayer = window.dataLayer || [];
            function gtag(){dataLayer.push(arguments);}
            gtag('js', new Date());

            gtag('config', 'G-DWKX9XJ8QW');
        </script>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
        <meta name="description" content="" />
        <meta name="author" content="" />
        <title>Graham Delafield</title>
        <link rel="icon" type="image/x-icon" href="assets/img/favicon.ico" />
        <!-- Font Awesome icons (free version)-->
        <script src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" crossorigin="anonymous"></script>
        <script src="https://kit.fontawesome.com/1780c09f7a.js" crossorigin="anonymous"></script>
        <!-- Google fonts-->
        <link href="https://fonts.googleapis.com/css?family=Saira+Extra+Condensed:500,700" rel="stylesheet" type="text/css" />
        <link href="https://fonts.googleapis.com/css?family=Muli:400,400i,800,800i" rel="stylesheet" type="text/css" />
        <!-- Core theme CSS (includes Bootstrap)-->
        <link href="css/styles.css" rel="stylesheet" />
    </head>
    <body id="page-top">
        <!-- Navigation-->
        <nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Graham Delafield</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/img/profile.jpg" alt="Picture of Graham Delafield" /></span>
            </a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
            <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav">
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#about">About</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#employment">Employment</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#experience">Experience</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#education">Education</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#publications">Publications</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#funding">Funding</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#skills">Skills</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#awards">Awards</a></li>
                    <li class="nav-item"><a class="nav-link js-scroll-trigger" href="portfolio.html">Portfolio</a></li>
                </ul>
            </div>
        </nav>
        <!-- Page Content-->
        <div class="container-fluid p-0">
            <!-- About-->
            <section class="resume-section" id="about">
                <div class="resume-section-content">
                    <h1 class="mb-0">
                        Graham
                        <span class="text-primary">Delafield</span>, Ph.D.
                    </h1>
                    <div class="subheading mb-5">
                        228 Duncan Street · San Francisco, CA 94131 · (405) 219-4741 ·
                        <a href="mailto:graham.delafield@gmail.com" style="color: #3a426b;">graham.delafield@gmail.com</a>
                    </div>
                    <p class="lead mb-5">
                        I am a developing researcher in the fields of proteomics, bioinformatics, and computer science. 
                        Currently I am focused on analytical innovation and computational modeling to aid in the profiling and 
                        characterization of the human glycoproteome, performing graduate research at the University of Wisconsin-Madison.
                        Anticipating graduation in March, 2023, I am interested in gaining experience in industrial and governmental settings
                        that allow me to exercise my technical expertise and strong communication and leadership skills.
                    </p>
                    <div class="row">
                        <div class="social-icons">
                            <a class="social-icon" href="https://www.linkedin.com/in/daniel-graham-delafield-42531871/"><i class="fab fa-linkedin-in"></i></a>
                            <a class="social-icon" href="https://github.com/grahamdelafield"><i class="fab fa-github"></i></a>
                            <a class="social-icon" href="https://www.researchgate.net/profile/Daniel_Delafield"><i class="fab fa-researchgate"></i></i></a>
                            <a class="social-icon" href="https://scholar.google.com/citations?user=uuIHgqMAAAAJ&hl=en"><i class="fas fa-graduation-cap"></i></i></a>
                        </div>
                    </div>
                    <div class="row">
                        <button type="button" class="btn btn-primary">
                            <a class="button-text" href="assets/DanielGDelafield_Resume.pdf" download="DanielGDelafield_Resume">
                            My Resume
                            </a>
                        </button>
                        <button type="button" class="btn btn-primary" style="margin-left: 2rem;">
                            <a class="button-text" href="assets/CV_GrahamDelafield.pdf" download="CV_GrahamDelafield">
                            Download CV
                            </a>
                        </button>
                        <button type="button" class="btn btn-primary" style="margin-left: 2rem;">
                            <a class="button-text" href="https://www.lilabs.org">
                            My Research Group
                            </a>
                        </button>
                    </div>
                </div>
            </section>
            <!-- Employment-->
            <section class="resume-section" id="employment">
                <div class="resume-section-content">
                    <h2 class="mb-5">Employment</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Calico</h3>
                            <div class="subheading mb-3">Scientist</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Coming soon!
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">Employment Commences July 2023</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Genentech, BAS-O3M</h3>
                            <div class="subheading mb-3">Research Intern</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Developed custom, full-stack desktop applications to automate and streamline immunogenicity
                                    assessment of drug candidates, enabling >99% reduction in analysis time.
                                </li>
                                <li class="exp-list-item">
                                    Engineered custom user interfaces to enable high-throughput analysis of large proteomic datasets, providing
                                    a 500-fold increase in data utilization.
                                </li>
                                <li class="exp-list-item">
                                    Constructed analysis pipelines leading to the discovery of constitutive immunogenic signatures that enable 
                                    reliable quality control in immunopeptidomic workflows.  
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">June 2021 - August 2021</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Wisconsin-Alumni Research Foundation</h3>
                            <div class="subheading mb-3">Graduate Ambassador</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Identified novel scientific intellectual property from with a personal academic network of 300+ graduate 
                                    students and 50+ principal investigators.
                                </li>
                                <li class="exp-list-item">
                                    Implemented new outreach events, canvas campaigns, and network opportunities aimed towards educating investigators 
                                    over intellectual propoerty protection, management, and licensing.
                                </li>
                                <li class="exp-list-item">
                                    Augmented the $3 Billion+ WARF portfolio through addition of novel gene edited cell line technologies. 
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2018 - April 2023</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!--Experience-->
            <section class="resume-section" id="experience">
                <div class="resume-section-content">
                    <h2 class="mb-5">Research Experience</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Graduate Research Assistant</h3>
                            <div class="subheading mb-3">University of Wisconsin-Madison, PI: Lingjun Li</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Fabricated and validated PGC-enabled nano separation methodologies that provide up to 40% greater 
                                    protein, peptide, and glycopeptide identifications.
                                </li>
                                <li class="exp-list-item">
                                    Identified chromatographic conditions most suitable for glycopeptide label-free quantitation, isomeric 
                                    resolution, and MS-based profiling depth.
                                </li>
                                <li class="exp-list-item">
                                    Utilized DIA-MS to identify 1,245 dysregulated proteins across six diagnostic patterns correlated 
                                    with early-, mid-, and late-stage prostate cancer.
                                </li>
                                <li class="exp-list-item">
                                    Coupled machine learning with DIA-MS to quantify >9,300 cerebrospinal fluid proteins and elucidate 
                                    1,642 putative neurological disease markers across patient cohorts.
                                </li>
                                <li class="exp-list-item">
                                    Engineered and deployed custom web-application based solutions for browsing, filtering, and sharing 
                                    mass spectrometry data.
                                </li>
                                <li class="exp-list-item">
                                    Performed custom informatic analysis of MALDI-MSI datasets to normalize multiplexed datasets, extract 
                                    features of interest, and perform machine learning classification.
                                </li>
                                <li class="exp-list-item">
                                    Development of novel online multiplexed analysis of bottom-up, middle-down, and top-down
                                    glycoconjugate species utilizing capillary electrophoresis, porous graphitized carbon liquid
                                    chromatography, ion mobility spectrometry, and mass spectrometry.
                                </li>
                                <li class="exp-list-item">
                                    Leveraged custom liquid chromatography, capillary electrophoresis, and ion mobility to identify multiple 
                                    conformations of active biomolecules.
                                </li>
                                <li class="exp-list-item">
                                    Trained and led small research teams of new graduate students and non-expert scientists in the areas of 
                                    biomolecular analysis, bioinformatic processing, and project development.
                                </li>
                                <li class="exp-list-item">
                                    Maintained, serviced, and assisted in troubleshooting a variety of high-end mass spectrometers from several 
                                    major vendors (Thermo, Bruker, Agilent).
                                </li>
                            </ul>
                            <p></p>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2018 - June 2023</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Graduate Research Assistant</h3>
                            <div class="subheading mb-3">University of Oklahoma, PI: Si Wu</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Developed a three-dimensional method of glycopeptide purification and characterization 
                                    allowing universal application, unbiased separation, and rapid recognition through the 
                                    use of HPLC, concurrent fractionation, Mass Spectrometry, and Differential Ion Mobility 
                                    (FAIMS).
                                </li>
                                <li class="exp-list-item">
                                    Demonstrated correlation between differential ion mobility detection and glycopeptide 
                                    analytes based on backbone variation, glycan composition, and a combination thereof.
                                </li>
                                <li class="exp-list-item">
                                    Analyzed post translationally modified glycoproteins of control and SLE patient serum 
                                    immunoglobulin searching for correlations to immune response and glycan features based 
                                    on our previous glycopeptide purification platform.
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2016 - May 2018</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Honors Undergraduate Research Assistant</h3>
                            <div class="subheading mb-3">University of Oklahoma, PI: Si Wu</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Conceptualized, designed, and constructed an online affinity capture technique for 
                                    immunoprecipitation application involving intensive immunoglobulin purification, original 
                                    apparatus design, and top-down mass spectrometry.
                                </li>
                                <li class="exp-list-item">
                                    ·	Determined activity of novel biomass degrading enzymes from Great Lakes fungal samples 
                                    through culturing, secretome extraction, enzymatic assays, concentration determination, 
                                    electrophoresis, liquid chromatography and mass spectrometry.
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2015 - May 2016</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Education-->
            <section class="resume-section" id="education">
                <div class="resume-section-content">
                    <h2 class="mb-5">Education</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Doctor of Philosophy</h3>
                            <div class="subheading mb-3">University of Wisconsin-Madison</div>
                            <ul class="exp-list-head">
                                <li class="ed-list-item">
                                    Primary field: Chemistry
                                </li>
                                <li class="ed-list-item">
                                    Division: Analytical Chemistry
                                </li>
                                <li class="ed-list-item">
                                    GPA: 4.0
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2018 - March 2023</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Master of Science</h3>
                            <div class="subheading mb-3">University of Oklahoma</div>
                            <ul class="exp-list-head">
                                <li class="ed-list-item">
                                    Primary field: Chemistry
                                </li>
                                <li class="ed-list-item">
                                    Division: Analytical Chemistry
                                </li>
                                <li class="ed-list-item">
                                    GPA: 3.9
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2016 - May 2018</span></div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Bachelor of Science</h3>
                            <div class="subheading mb-3">University of Oklahoma</div>
                            <ul class="exp-list-head">
                                <li class="ed-list-item">
                                    Major: Biochemistry
                                </li>
                                <li class="ed-list-item">
                                    Minor: Music
                                </li>
                                <li class="ed-list-item">
                                    Honors: Cum Laude
                                </li>
                            </ul>
                        </div>
                        <div class="flex-shrink-0"><span class="text-primary">August 2011 - May 2016</span></div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Publications-->
            <section class="resume-section" id="publications">
                <div class="resume-section-content">
                    <h2 class="mb-5" style="margin-bottom: 1rem;">Publications</h2>
                    <div class="row">
                        <button type="button" class="btn btn-primary" style="margin-top: 0rem; margin-bottom: 2rem; margin-left: 1rem;">
                            <a class="button-text" href="portfolio.html#publications">
                            See Full Portfolio
                            </a>
                        </button>
                    </div>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, H.N. Miles, T.N. Liu, W.A. Ricke, L. Li. “Inclusion of Porous Graphitic Carbon Chromatography Yields Greater 
                        Protein Identification, Compartment and Process Coverage, and Enables More Reflective Protein-Level Label-Free Quantitation.” Published 
                        online October 10, 2023. <a href="https://doi.org/10.1021/acs.jproteome.3c00373">10.1021/acs.jproteome.3c00373</a>
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, H.N. Miles, W.A. Ricke, L. Li (2022). “Higher Temperature Porous Graphitic Carbon Separations 
                        Differentially Impact Distinct Glycopeptide Classes.” <i>Journal for the American Society for Mass Spectrometric</i>. Published online
                        November 30, 2022. <a href="https://doi.org/10.1021/jasms.2c00249">10.1021/jasms.2c00249</a>
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, C. Kaminsky, G. Liu, L. Li. (2022) “High-End Ion Mobility Mass Spectrometry:  A Current Review of Analytical 
                        Capacity in Omics Applications and Structural Investigations.” <i>Trends in Analytical Chemistry.</i> Published online August 24, 2022.
                        <a href="https://doi.org/10.1016/j.trac.2022.116761">10.1016/j.trac.2022.116761</a>
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, H.N. Miles, Y. Liu, W.A. Ricke, L. Li, (2022). "Complementary Proteome and Glycoproteome Access Revealed 
                        Through Comparative Analysis of Reversed Phase and Porous Graphitic Carbon Chromatography." <i>Anal. Bioanal. Chem.</i> Published 
                        online February 9, 2022. <a href="https://doi.org/10.1007/s00216-022-03934-7" class="href">10.1007/s00216-022-03934-7</a>
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield†</b>, H.N. Miles†, L. Li, (2021). "Recent Developments and Applications of Quantitative Proteomics Strategies for High-Throughput 
                        Cancer Biomolecular Analyses in Cancer Research." <i>RSC Chemical Biology</i>. Published online May 15, 2021. <a href="http://dx.doi.org/10.1039/D1CB00039J">
                        10.1039/D1CB00039J</a> (†Co-first authors)
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, L. Li, (2020). "Recent Advances in Analytical Approaches for Glycan and Glycopeptide Quantitation." 
                        <i>Molecular & Cellular Proteomics</i>. Published online June 23, 2020. <a href="https://doi.org/10.1074/mcp.R120.002095">
                        10.1074/mcp.R120.002095</a>
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, H.N. Miles, W.A. Ricke, L. Li. "Proteomic Fingerprinting of Prostate Cancer Progression Through Library-Free DIA-MS Reveals 
                        Systematic and Conserved Pathway Dysregulation." (In Submission) <a href="https://grahamdelafield.github.io/assets/img/Posters/USHUPO_2023.png">Poster Available</a>
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield†</b>, X. Zhong†, L. Li. "Sample Agnostic Spectral Libraries Enable Quantitation of >9,300 Cerebrospinal 
                        Fluid Proteins Across Neurodegenerative Disease Patient Cohorts." (In Submission) († Co-First Authors) <a href="https://grahamdelafield.github.io/assets/img/Posters/ChemPoster2022.png">
                        Poster Available</a>
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, H.N. Miles, W.A. Ricke, L. Li. “Six-plex Isobaric Labeling Reveals Glycosylation Dysregulation Across 
                        Benign, Metastatic, and Tumorigenic Prostate Cancer.” (In Preparation)
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield</b>, N. Treuttner, H.N. Miles, W.A. Ricke, L. Li.  “Capsule Network Retention Time Correction Facilitates 
                        Prostatic Tumorigenesis Discovery Through Porous Graphitic Carbon DIA-MS.” (In Preparation)
                    </p>
                    <p class="pub-text">
                        <b>D.G. Delafield†</b>, X. Zhong†, L. Li. "Enhanced Proteomic Coverage Enabled through BoxCar DIA and Sample Agnostic Spectral 
                        Libraries." (In Preparation) († Co-First Authors)
                    </p>
                    <p class="pub-text">
                        Y. Liu, <b>D.G. Delafield</b>, L. Li "Comprehensive Mass Spectrometric Characterization of Neuropeptidome in 
                        Nervous System of the Atlantic Blue Crab, Callinectes sapidus." <i>Analytical Chemistry</i>. (To Be Submitted).
                    </p>
                    <p class="pub-text">
                        S. Xu, Z. Zhu, <b>D.G. Delafield</b>, M.J. Rigby, G. Lu, L. Puglielli, L. Li, "Spatially and Temporally Probing 
                        Distinctive Glycerophospholipid Alteration In Alzheimer's Disease Mouse Brain Via High-Resolution Ion Mobility-Enabled
                        sn-Position-Resolved Lipidomics." In Submission.
                    </p>
                    <p class="pub-text">
                        S. Xu, Z. Zhu, T.J. Gu, Z. Wang, <b>D.G. Delafield</b>, M.J. Rigby, G. Lu, M. Ma, P. Liu, L. Puglielli, L. Li, "sn-position Resolved Quantification 
                        of Aminophospholipids in Alzheimer's Disease Progression by Isotopic N, N-dimethyl leucine Labeling and High-resolution Ion Mobility Mass Spectrometry." 
                        In Submission.
                    </p>
                    <p class="pub-text">
                        M. Ma, Q. Yu, <b>D.G. Delafield</b>, Y. Cui, Z. Li, W. Wu, X. Shi, A. Gutierrez, P.R. Westmark, M. Xu, C.J. Westmark, L. Li (2023) 
                        “On-tissue Spatial Proteomics Integrating MALDI-MS Imaging with Shotgun Proteomics Reveals Soy Consumption-induced Biomarkers in a 
                        Fragile X Syndrome Mouse Model.” Publishbed online December 18, 2023. <a href="https://pubs.acs.org/doi/full/10.1021/acschemneuro.3c00497">10.1021/acschemneuro.3c00497</a>
                    </p>
                    <p class="pub-text">
                        Z. Zhu, S. Xu, Z. Wang, <b>D.G. Delafield</b>, M.J. Rigby, G. Lu, T. Gu, P. Liu, M. Ma, L. Puglielli, L. Li, (2023)
                        "Chiral Pair Isobaric Labeling Strategy for Multiplexed Absolute Quantitation (CHRISTMAS) Enabled Quantitatively 
                        Probing of Enantiomeric Amino Acids Alteration in Alzheimer's Disease Progression," <i>Analytical Chemistry</i>.
                        Published online November 30, 2023. <a href="https://doi.org/10.1021/acs.analchem.3c03847">10.1021/acs.analchem.3c03847</a>
                    </p>
                    <p class="pub-text">
                        G. Li, C.K. Jeon, M. Ma, Z. Zheng, Y. Jia, Z. Zheng, <b>D.G. Delafield</b>, G. Lu, E. Romanova, J. Sweedler, B. Ruotolo, L. Li, (2023)
                        "Site-specific Chirality-conferred Structural Compaction Differentially Rescues the Cytotoxicity of Aβ42." <i>Chemical 
                            Science</i>. Published online May 8, 2023. <a href="https://pubs.rsc.org/en/content/articlelanding/2023/sc/d3sc00678f">10.1039/D3SC00678F</a>
                    </p>
                    <p class="pub-text">
                        H. Zhang, <b>D.G. Delafield</b>, L. Li (2023) "Mass spectrometry imaging: the rise of spatially resolved single-cell omics," <i>Nature Methods</i> 
                        20, 327-330. <a href="https://www.nature.com/articles/s41592-023-01774-6">10.1038/s41592-023-01774-6</a>
                    </p>
                    <p class="pub-text">
                        Y Shi, Z. Li, B. Wang, Z. Ye, H. Ye, <b>D.G. Delafield</b>, X. Shi, Z. Chen, F. Ma, L. Li (2022) "Enabling Global Analysis of Protein 
                        Citrullination and Homocitrullination via Biotin Thiol Tag-Assisted Mass Spectrometry." <i>Analytical Chemistry</i>. Published online 
                        December 13, 2022. <a href="https://doi.org/10.1021/acs.analchem.2c03844">10.1021/acs.analchem.2c03844</a>.
                    </p>
                    <p class="pub-text">
                        N. Wang, N. Wang, S. Yu, H. Zhang, S. Tang, D. Wang, W. Lu, H. Li, <b>D.G. Delafield</b>, 
                        Y. Kong, X. Wang, C. Shao, L. Lv, G. Wang, R. Tan, N. Wang, H. Hao, H. Ye (2022) “Cyclic Immonium Ion 
                        of Lactyllysine Reveals Widespread Lactylation in the Human Proteome.” <i>Nature Methods</i>. Published online June 28, 2022.
                        <a href="https://www.nature.com/articles/s41592-022-01523-1">10.1038/s41592-022-01523-1</a>
                    </p>
                    <p class="pub-text">
                        Z. Li, D. M. Tremmel, F. Ma, Q. Yu, M. Ma, <b>D.G. Delafield</b>, Y. Shi, B. Wang, S. A. Mitchell, A. K. Feeney,
                        V. S. Jain, S. D. Sackett, J. S. Odorico, L. Li (2020) "Proteome-wide and Matrisome-specific Alterations 
                        during Human Pancreas Development and Maturation" <i>Nature Communications</i>. Published online February 15, 2021.
                        <a href="https://www.nature.com/articles/s41467-021-21261-w">10.1038/s41467-021-21261-w</a>
                    </p>
                    <p class="pub-text">
                        Ruiz, M., Y. Yang, C. A. Lochbaum, <b>D.G. Delafield</b>, J. J. Pignatello, L. Li and J. A. Pedersen (2019).
                        "Peroxymonosulfate Oxidizes Amino Acids in Water without Activation." <i>Environmental Science and 
                        Technology</i>. 53, 10845-10854. <a href="https://pubs.acs.org/doi/abs/10.1021/acs.est.9b01322">
                        10.1021/acs.est.9b01322</a>
                    </p>
                    <p class="pub-text">
                        Li, G., <b>D.G. Delafield</b> and L. Li (2019). "Improved Structural Elucidation of Peptide Isomers and
                        Their Receptors Using Advanced Ion Mobility-Mass Spectrometry." <i>Trends in Analytical Chemistry</i>. 
                        Published online, June 4, 2019, in press. <a href="https://doi.org/10.1016/j.trac.2019.05.048">
                        10.1016/j.trac.2019.05.048</a>
                    </p>
                    <p class="pub-text">
                        Ma, H., <b>D.G. Delafield</b>, Z. Wang, J. You and S. Wu (2017). "Finding Biomass Degrading Enzymes 
                        Through an Activity-Correlated Quantitative Proteomics Platform (ACPP)." <i>Journal of American 
                        for Mass Spectrometry</i>. 28, 655-663. <a href="https://pubs.acs.org/doi/abs/10.1021/jasms.8b05509">
                        10.1021/jasms.8b05509</a>
                    </p>
                </div>
            </section>
            <!--Funding-->
            <hr class="m-0" />
            <section class="resume-section" id="funding">
                <div class="resume-section-content">
                    <h2 class="mb-5">Funding</h2>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">DiLeu-enabled multiplexed quantitation for biomarker discovery and validation in Alzheimer's disease (Renewal)</h3>
                            <div class="subheading mb-3">2R01AG052324-02</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Funding Awarded: $2,835,460
                                </li>
                                <li class="exp-list-item">
                                    Project Length: 5 years
                                </li>
                                <li class="exp-list-item">
                                    Role: Contributing Author
                                </li>
                                <li class="exp-list-item">
                                    Award Date: May 2023
                                </li>
                            </ul>
                        </div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Creating a region- specific biomolecular atlas of the brain of Alzheimer’s disease</h3>
                            <div class="subheading mb-3">1R01AG078794-01</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Funding Awarded: $3,780,730
                                </li>
                                <li class="exp-list-item">
                                    Project Length: 5 years
                                </li>
                                <li class="exp-list-item">
                                    Role: Contributing Author
                                </li>
                                <li class="exp-list-item">
                                    Award Date: September 2022
                                </li>
                            </ul>
                        </div>
                    </div>
                    <div class="d-flex flex-column flex-md-row justify-content-between mb-5">
                        <div class="flex-grow-1">
                            <h3 class="mb-0">Acquisition of a Dual-Source, High-Performance, Ion Mobility, Quadrupole Time-of-Flight Mass Spectrometry System for Biomedical Research at UW-Madison</h3>
                            <div class="subheading mb-3">1S10OD028473-01A1</div>
                            <ul class="exp-list-head">
                                <li class="exp-list-item">
                                    Funding Awarded: $1,275,704
                                </li>
                                <li class="exp-list-item">
                                    Project Length: 1 year, direct
                                </li>
                                <li class="exp-list-item">
                                    Role: <b>Primary Author</b>
                                </li>
                                <li class="exp-list-item">
                                    Award Date: May 2021
                                </li>
                            </ul>
                        </div>
                    </div>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Skills-->
            <section class="resume-section" id="skills">
                <div class="resume-section-content">
                    <h2 class="mb-5">Skills</h2>
                    <div class="subheading mb-3">Proteomics Research</div>
                    <ul class="fa-ul mb-3">
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            High-Throughput Mass Spectrometry
                            <li>
                                <span class="fa-li-low"></span>
                                Instrumentation: Thermo Elite, Q-Exactive Series | Agilent 6560 IM-QTOF | Bruker timsTOF Pro | Waters Synapt G2 
                            </li>
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Multi-tier Protein Analysis
                            <li>
                                <span class="fa-li-low"></span>
                                Bottom-up | middle-down | top-down 
                            </li>
                            <li>
                                <span class="fa-li-low"></span>
                                Protein & peptide identification | Glycan and glycopeptide characterization  
                            </li>
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            High Pressure Liquid Chromatography
                            <li>
                                <span class="fa-li-low"></span>
                                Nano & Standard Flow | Reversed-phase | Porous graphitic carbon | Ion-Exchange | Size Exclusion
                            </li>
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Capillary Electrophoresis
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-check"></i></span>
                            Custom Bioinformatics and Programming
                        </li>
                    </ul>
                    <div class="subheading mb-3">Programming Languages & Tools</div>
                    <ul class="list-inline dev-icons">
                        <li class="list-inline-item"><i class="fab fa-python"></i></li>
                        <li class="list-inline-item"><i class="fab fa-r-project"></i></li>
                        <li class="skills-image"><img src="./assets/img/Clang.svg" alt="C Language"></li>
                        <li class="list-inline-item"><i class="fa-brands fa-rust"></i></li>
                        <li class="list-inline-item"><i class="fab fa-html5"></i></li>
                        <li class="list-inline-item"><i class="fab fa-css3-alt"></i></li>
                        <li class="list-inline-item"><i class="fab fa-js-square"></i></li>
                        <li class="list-inline-item"><i class="fab fa-bootstrap"></i></li>
                    </ul>
                </div>
            </section>
            <hr class="m-0" />
            <!-- Awards-->
            <section class="resume-section" id="awards">
                <div class="resume-section-content-sh">
                    <h2 class="mb-5">Awards & Certificates</h2>
                    <ul class="fa-ul mb-0">
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Gary Parr Memorial Award for Outstanding Bioanalytical Graduate Research (2023)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Luminex Summer Fellowship, Department of Chemistry (2022)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Department of Chemistry Research Travel Award (2022)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Harold Hay Fellowship, Department of Chemistry (2022)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Student Research Travel Award (2019)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Honored Instructor Awward (2018)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            ASMS Travel Grant (2018, 2019)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Head Teaching Assistant (2017)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Certifcate of Distinction in Teaching (2017, 2018)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            College of Arts & Sciences Travel Grant (2017)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Scott Liang Outstanding Undergraduate Research Award (2016)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            J. Lee Burke Outstanding Student Achievement Award (2016)
                        </li>
                        <li>
                            <span class="fa-li"><i class="fas fa-trophy text-warning"></i></span>
                            Outstanding Senior Man Award (2016)
                        </li>
                    </ul>
                </div>
            </section>
        </div>
        <!-- Bootstrap core JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.bundle.min.js"></script>
        <!-- Third party plugin JS-->
        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js"></script>
        <!-- Core theme JS-->
        <script src="js/scripts.js"></script>
    </body>
</html>
